src/dal-plugin/Stream.mm

```diff
@@ -107,9 +111,23 @@ - (CFTypeRef)clock {
     return _clock;
 }
 
+- (NSSize)testCardImageSize {
```

Comment:
Oh sorry I didn't catch this the first time. I'd just name this `testCardSize` (making it the accessor for the `testCardSize` property)